src/openms/CMakeLists.txt

```diff
@@ -93,7 +93,9 @@ set(OPENMS_DEP_LIBRARIES  ${COIN_LIBRARIES}
                           ${BZIP2_LIBRARIES}
                           ${ZLIB_LIBRARIES}
                           ${SQLite_LIBRARY}
+                          ${HDF5_CXX_LIBRARIES}
                           ${GLPK_LIBRARIES}
+                          ${CMAKE_DL_LIBS}
```

Comment:
strange, but this fixed the openms_pr_ci/compiler=g++-7,os_label=ubuntu_1804 builds...